Gem/Code/Source/UserSettings/MultiplayerSampleUserSettings.cpp

```diff
@@ -22,6 +25,15 @@
 
 namespace MultiplayerSample
 {
+    static constexpr const char* DefaultGraphicsApi = "";       // default to the platform-specific default graphics API
```

Comment:
FTW!

Reply:
Thanks for suggesting this on a previous PR! :)